src/scr.c

```diff
@@ -678,18 +678,32 @@ static int scr_get_params()
   /* if not enabled, bail with an error */
   if (! scr_enabled) {
     scr_param_finalize();
+    if (scr_my_rank_world == 0) {
+      scr_dbg(1, "SCR_ENABLE=%d", scr_enabled);
+    }
```

Comment:
Let's pull this out from the if block and move it up to line 677.

Reply:
new updated is submitted based on the comments.